vllm/attention/backends/flash_attn.py

```diff
@@ -494,6 +494,29 @@ def _add_seq_group(
                                  seq_len, context_len, start_idx,
                                  self.block_size, inter_data.block_tables)
 
+    def _use_graph_block_tables(self, num_seqs: int,
```

Comment:
refactor

Reply:
Suggest naming: `_get_block_table_with_cuda_graph`

Reply:
I gave it some thought - `_get_block_table_with_cuda_graph` seems to suggest that cuda-graphs are a property of the block-tables. while we simply copy the python block tables to the block-table tensor in `self.runner.graph_block_tables`.
 
IMHO `_use_graph_block_tables` captures the intent, given `self.runner.graph_block_tables` is the tensor being filled.

Perhaps, `_prepare_graph_block_tables` / `_get_graph_runner_block_tables`  is a better alternative. 

What do you think ? Happy to make the change if you think otherwise.

Reply:
_get_graph_runner_block_tables looks better to me.

Reply:
Changed it to `_get_graph_runner_block_tables`. Thanks.